VAMobile/src/screens/HealthScreen/SecureMessaging/ViewMessage/CollapsibleMessage.tsx

```diff
@@ -86,7 +86,7 @@ function CollapsibleMessage({ message, isInitialMessage, collapsibleMessageRef }
               mx={theme.dimensions.gutter}
               mt={theme.dimensions.contentMarginTop}
               mb={theme.dimensions.contentMarginBottom}>
-              <LoadingComponent text={t('secureMessaging.viewMessage.loadingAttachment')} inlineSpinner={true} />
```

Comment:
this would essentially wrap these pre-existing in a scrollview. I'm not positive that's a problem but I just want to hear you tinkered and made sure.

Reply:
Mentioned the issue with the scrollview in this [comment](https://github.com/department-of-veterans-affairs/va-mobile-app/pull/8098/files#r1513638975)